src/FirebaseAuth.js

```diff
@@ -303,24 +302,16 @@
      * wish to log in as the newly created user, call $authWithPassword() after the promise for
      * this method has been resolved.
      *
-     * @param {Object|string} emailOrCredentials The email of the user to create or an object
-     * containing the email and password of the user to create.
-     * @param {string} [password] The password for the user to create.
+     * @param {Object} credentials An object containing the email and password of the user to create.
      * @return {Promise<Object>} A promise fulfilled with the user object, which contains the
      * uid of the created user.
      */
-    createUser: function(emailOrCredentials, password) {
+    createUser: function(credentials) {
       var deferred = this._q.defer();
 
-      // Allow this method to take a single credentials argument or two separate string arguments
-      var credentials = emailOrCredentials;
-      if (typeof emailOrCredentials === "string") {
-        this._log.warn("Passing in credentials to $createUser() as individual arguments has been deprecated in favor of a single credentials argument. See the AngularFire API reference for details.");
-
-        credentials = {
-          email: emailOrCredentials,
-          password: password
-        };
+      // Throw an error if they are trying to pass in separate string arguments
+      if (typeof credentials === "string") {
+        throw new Error("Passing in credentials to $createUser() as individual arguments has been removed in favor of a single credentials argument. See the AngularFire API reference for details.");
```

Comment:
This could be ambiguous, as the context assumes one is upgrading and has read the old API docs. Maybe we should just say "$createUser() expects an object containing `email` and `password` arguments, but received a string."? Same suggestion for below.